kilkari/src/main/java/org/motechproject/nms/kilkari/util/Timer.java

```diff
@@ -0,0 +1,64 @@
+package org.motechproject.nms.kilkari.util;
```

Comment:
should we put this in a more common package for use with other modules for perf testing?